apps/moderation/tests/test_serializers.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+
+from apps.moderation.serializers import ModeratedModelSerializer
+from apps.search.documents import instant_search_documents_map
+from core.fields import HTMLField
+
+from .moderated_serializers import MODERATED_SERIALIZERS as serializers
+
+
+@pytest.mark.django_db()
+class TestModeratedSerializers:
+
+    # allow these fields to have choices instead of instant search
+    instant_search_field_exceptions = ['birth', 'death']
+
+    @classmethod
```

Comment:
Why class methods?

Reply:
IDE kept saying it should be classmethod since they're not using anything self. Should I change?

Reply:
Gotcha. No need to change it )